docs/notebooks/jax-101/01-jax-basics.md

```diff
@@ -0,0 +1,375 @@
+---
+jupytext:
+  formats: ipynb,md:myst
+  text_representation:
+    extension: .md
+    format_name: myst
+    format_version: 0.13
+    jupytext_version: 1.10.0
+kernelspec:
+  display_name: Python 3
+  name: python3
+---
+
++++ {"id": "6_117sy0CGEU"}
+
+# JAX as accelerated Numpy
+
+[![Open in Colab](https://colab.research.google.com/assets/colab-badge.svg)](https://colab.research.google.com/github/google/jax/blob/master/docs/notebooks/jax-101/01-jax-basics.ipynb)
+
+In this first colab you will learn the very fundamentals of JAX.
+
++++ {"id": "CXjHL4L6ku3-"}
+
+## Getting started with JAX numpy
+
+Fundamentally, JAX is a library that enables transformations of array-manipulating programs written with a Numpy-like API. 
+
+Over the course of this series of guides, we will unpack exactly what that means. For now, you can think of JAX as *differentiable Numpy that runs on accelerators*.
+
+The code below shows how to import JAX and create a vector.
+
+```{code-cell} ipython3
+:id: ZqUzvqF1B1TO
+
+import jax
+import jax.numpy as jnp
+
+x = jnp.arange(10)
+print(x)
+```
+
++++ {"id": "rPBmlAxXlBAy"}
+
+So far, everything is just like Numpy. A big appeal of JAX is that you don't need to learn a new API. Many common Numpy programs would run just as well in JAX if you substitute `np` for `jnp`. However, there are some important incompatibilities which we touch on at the end of this colab.
+
+You can notice the first difference if you check the type of `x`. It is a variable of type `DeviceArray`, which is the way JAX represents arrays.
+
+```{code-cell} ipython3
+:id: 3fLtgPUAn7mi
+
+x
+```
+
++++ {"id": "Yx8VofzzoHFH"}
+
+One useful feature of JAX is that the same code can be run on different backends -- CPU, GPU and TPU.
+
+We will now perform a dot product to demonstrate that it can be done in different devices without changing the code. We use `%timeit` to check the performance. 
+
+(Technical detail: when a JAX function is called, the corresponding operation is dispatched to an accelerator to be computed asynchronously when possible. The returned array is therefore not necessarily 'filled in' as soon as the function returns. Thus, if we don't require the result immediately, the computation won't block Python execution. Therefore, unless we `block_until_ready`, we will only time the dispatch, not the actual computation. See [Asynchronous dispatch](https://jax.readthedocs.io/en/latest/async_dispatch.html#asynchronous-dispatch) in the JAX docs.)
+
+```{code-cell} ipython3
+:id: mRvjVxoqo-Bi
+
+long_vector = jnp.arange(int(1e7))
+
+%timeit jnp.dot(long_vector, long_vector).block_until_ready()
+```
+
++++ {"id": "DKBB0zs-p-RC"}
+
+Tip: try running the code above twice, once without an accelerator, and once with a GPU runtime (use *Runtime→Change Runtime Type* and choose `GPU`). Notice how it runs much faster on GPU.
+
++++ {"id": "PkCpI-v0uQQO"}
+
+## JAX first transformation: Grad
```

Comment:
Suggestion: 

- Since it's technically `grad` (or `jax.grad`), would

```suggestion
## JAX first transformation: `grad`
```

(or "JAX's..." or I'm overthinking)

make more sense?